doc/source/contributing.rst

```diff
@@ -701,6 +701,18 @@ Tests that we have ``parametrized`` are now accessible via the test name, for ex
    test_cool_feature.py::test_dtypes[int8] PASSED
    test_cool_feature.py::test_series[int8] PASSED
 
+Sometimes, it does make sense to bundle test functions together into a single class, either because the test file is testing multiple functions from a single module, and
```

Comment:
actually, let's split this out to a section after 'Writing Tests' and before 'Parametrize' (you may want to move some prose from Parametrize that is related to this as well).